test/source/tests/decrypt.ts

```diff
@@ -514,6 +515,26 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
       expect(urls.length).to.be.lessThan(2);
     }));
 
+    ava.default('decrypt - print feature in pgp block', testWithBrowser('compatibility', async (t, browser) => {
+      const threadId = '182917712be838e1';
+      const acctEmail = 'flowcrypt.compatibility@gmail.com';
+      const inboxPage = await browser.newPage(t, TestUrls.extension(`chrome/settings/inbox/inbox.htm?acctEmail=${acctEmail}&threadId=${threadId}`));
+      await inboxPage.waitAll('iframe');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
+      await pgpBlock.waitForSelTestState('ready');
+      expect(await pgpBlock.isElementPresent('@action-print')).to.be.true;
+      await pgpBlock.click('@action-print');
```

Comment:
It's be nice to use
```
const printPage = await browser.newPageTriggeredBy(t, () => pgpBlock.click('@action-print'));
```
instead of these 4 lines, but we'd need `<body data-test-view-state="loaded">` in `printPGPBlock`